h/app.py

```diff
@@ -113,6 +113,7 @@ def includeme(config):
     config.include('h.api', route_prefix='/api')
 
     # Core site modules
+    config.include('h.activity')
```

Comment:
This should go in the block below. Sorry, that's probably not super clear, but the top block is site-wide utilities, infrastructure, plumbing stuff. The bottom block is specific site modules.